lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -238,6 +244,47 @@ def _partition_remote(
 
         return elements
 
+    @staticmethod
+    def _partition_remote(
+        file: BinaryIO,
+        aryn_api_key: str,
+        aryn_partitioner_address=DEFAULT_ARYN_PARTITIONER_ADDRESS,
+        threshold: float = 0.4,
+        use_ocr: bool = False,
+        ocr_images: bool = False,
+        ocr_tables: bool = False,
+        extract_table_structure: bool = False,
+        extract_images: bool = False,
+        selection_size: int = 25,
+    ) -> List[Element]:
+        file.seek(0)
+        pdf_reader = PyPDF2.PdfReader(file)
```

Comment:
Do we need to bring in PyPDF2 for this? It looks like you can get teh number of pages with pdfminer as well, which we already import (see https://stackoverflow.com/a/47686921). We keep getting deprecation warnings about PyPDF2, so would be nice to avoid if its easy.

Reply:
Thanks for the optimization. Done.